scripts/appleseed.package.py

```diff
@@ -190,6 +191,103 @@ def merge_tree(src, dst, symlinks=False, ignore=None):
         raise Error(errors)
 
 
+def create_mac_icon(image, icon_path):
+    icon_set = "appleseed.iconset"
+    safe_make_directory(icon_set)
+
+    for size in (16, 32, 64, 128, 256, 1024, 2048):
+        if size < 1024:
+            name = "icon_{0}x{0}.png".format(size)
+            cmd = ['sips', '-z', str(size), str(size), image, '--out', os.path.join(icon_set, name)]
```

Comment:
Is `sips` readily available on a developer macOS machine?

Reply:
yes sips is available back to at least macOS snow leopard 10.6.3